packages/ripple-keypairs/src/secp256k1.ts

```diff
@@ -1,13 +1,12 @@
-import * as elliptic from 'elliptic'
-
 import Sha512 from './Sha512'
+import { secp256k1 } from '@noble/curves/secp256k1'
 
-const secp256k1 = elliptic.ec('secp256k1')
+const Zero = BigInt(0)
```

Comment:
nit: I think this library usually uses all caps for constants.

```suggestion
const ZERO = BigInt(0)
```